MockKit/MockService.swift

```diff
@@ -23,7 +23,10 @@ public final class MockService: Service {
     
     public var analytics: Bool
     
+    public let maxHistoryItems = 1000
+    
     public var history: [String]
+    private let historyQueue = DispatchQueue(label: "MockService_historyQueue")
```

Comment:
It looks like history is currently being accessed from UI code in MockServiceSettingsViewController.

Putting history writing onto a new thread wouldn't solve the issue, as you'd still have writing from one thread (historyQueue) and reading on another thread (main).

I'd recommend just dispatching the writes onto .main instead.

Reply:
Good point.  I see now that `history` is `public`.  I'll fix this to use locking.